desktop/core/src/desktop/views.py

```diff
@@ -125,9 +128,10 @@ def saml_login_headers(request):
   try:
     userprofile.update_data({'X-CSRF-TOKEN': request.META['CSRF_COOKIE']})
     userprofile.save()
-  except:
+  except Exception as e:
```

Comment:
Same as above

Reply:
@Harshg999 
This has been added as Ruff linter gave error 
`desktop/core/src/desktop/views.py:420:5: E722 Do not use bare `except``
Are we talking about removing the entire except statement i.e. line no 131 and 132?

Reply:
changed `except Exception as e`: to `except Exception`